client/lib/analytics/track-resurrections/index.jsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect } from '@wordpress/element';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import getUserSetting from 'calypso/state/selectors/get-user-setting';
+
+const TrackResurrections = ( { login_time, yearInterval, isFetchingUserSettings } ) => {
+	const isResurrected = ( loginTime, previousLoginTime ) => {
+		const differenceInMilliseconds = loginTime - previousLoginTime;
+		const differenceInYears = differenceInMilliseconds / ( 1000 * 60 * 60 * 24 * 365 );
```

Comment:
The calculation is ideal for a constant. We usually do them like this:

```
const YEAR_IN_MILLISECONDS = 1000 * 60 * 60 * 24 * 365;
```

and then we use it in the code.

It could greatly improve the readability of the code.